imports/plugins/included/ui-home/client/templates/about.html

```diff
@@ -0,0 +1,42 @@
+<template name="aboutTemplate"> 
+    <section id="about">
+      <div>
+        <div>
+          <h2 class="text-center about-text">ORIGINS</h2>
+    
+          <p style="text-align: center; line-height: 2.5em; width: 50%; margin: 0 auto;  font-family: 'Roboto', sans-serif; font-weight:100; font-size: 17px;">
```

Comment:
Why are we using inline style here. Can the styling not be placed in an appropriate `.less` file?